tests/unittests/MLTest.cpp

```diff
@@ -881,16 +881,18 @@ static void generateImageData(Tensor &images, Tensor &labels, PseudoRNG &PRNG) {
 
 /// Test the convolutional layer.
 /// Use a simple convnet to learn two classes of images: Line and Cross.
-TEST_P(MLTest, convNetForImageRecognition) {
+/// Add quantization support for the convolution layer test case
```

Comment:
Period at the end of the sentence. 

 Add quantization support for the convolution layer test case  ->  This test checks the results of the quantized network.

Reply:
done